docs/examples/ns1/k8gb-cluster-ns1-us-east-1.yaml

```diff
@@ -8,3 +8,5 @@ k8gb:
 
 ns1:
   enabled: true
+  # endpoint: https://ns1.example.com
+  # ignoreSSL: false
```

Comment:
`ignoreSSL` can stay `false` uncommented

Reply:
This way it will be set into deployment, which makes args list longer and hard to read. I'd leave defaults into provider's default settings

Reply:
No, in case of `false` the `if` in the template will not add the parameter at all

Reply:
@ytsarev amended

Reply:
@k0da but apparently not pushed?

Reply:
Sorry, missed that you referenced example values, while I've been amending default chart values. Fixed now